src/router/routes.ts

```diff
@@ -216,7 +216,7 @@ export const routes = [
     // must be last
     // NB: this route is IA-specific but App::fetchData() will reroute
     // if user is on a route not valid for the current filing type
-    path: '*',
+    path: '/:catchAll(.*)',
```

Comment:
This was causing an error. Reference: https://stackoverflow.com/questions/63526486/vue-router-catch-all-wildcard-not-working